### PR TITLE
Feature 48 devices edit device

### PR DESCRIPTION
**Pull request by gabagardi**

### CODE REVIEW

**Review by aliabdellatif0 — APPROVED** [review]

File: src/components/EditDeviceForm/EditDeviceForm.scss [hunk 1]
```diff
@@ -0,0 +1,125 @@
+@use "../../assets/styles/variables"as *;
+@use "../../assets/styles/mixins"as *;
+@use "../../assets/styles/typography"as *;
```
Comment (Halfiies) [inline src/components/EditDeviceForm/EditDeviceForm.scss:3]: i think we dont need this?

File: src/containers/Devices/Devices.scss [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+  .devices-page { 
+  
     height: 100vh;
-  }
-}
+    width: 95%;
+    margin: 7vh auto;
+  
+    @include mobile {   
+      display: none;
```
Comment (Halfiies) [inline src/containers/Devices/Devices.scss:11]: this is the problem line thats stoping devices showing on mobile

Comment (gabagardi) [reply src/containers/Devices/Devices.scss:11]: Will fix on a different ticket

File: src/components/EditDeviceForm/EditDeviceForm.jsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+const EditDeviceForm = (props) => {
+    const { device } = useParams();
+
+    const filteredDevice = devicesResponse.find(
+        (singleDevice) => singleDevice.name === device
+
+    );
+
+
```
Comment (Charlie-robin) [inline src/components/EditDeviceForm/EditDeviceForm.jsx:17]: If it doesn't find something redirect somewhere

**Review by Charlie-robin — CHANGES_REQUESTED** [review]

**Pull request merged**